docs/scripting/resources/weatherid.md

```diff
@@ -28,10 +28,10 @@ To be used with [SetWeather](../functions/SetWeather) and [SetPlayerWeather](../
 20 = UNDERWATER (greenish, foggy)
 ```
 
-There are 256 different weather IDs (0-255), values higher than 255 or lower than 0 are turned into remainder of the division by 256 (for example, weather ID 300 is the same as ID 44, because 300 % 256 = 44). Weather IDs 0-22 appear to work correctly but other IDs result in strange effects such as pink sky and flashing textures during certain times.
+There are 20 different weather IDs (0-20), however the game does not feature any range checking for weather IDs and thus you can use weather IDs upto 255.  Values higher than 255 or lower than 0 are turned into remainder of the division by 256 (for example, weather ID 300 is the same as ID 44, because 300 % 256 = 44). Weather IDs 0-22 appear to work correctly but other IDs result in strange effects such as pink sky and flashing textures during certain times.
```

Comment:
```suggestion
There are 21 different weather IDs (0-20), however the game does not feature any range checking for weather IDs and thus you can use weather IDs upto 255.  Values higher than 255 or lower than 0 are turned into remainder of the division by 256 (for example, weather ID 300 is the same as ID 44, because 300 % 256 = 44). Weather IDs 0-22 appear to work correctly but other IDs result in strange effects such as pink sky and flashing textures during certain times.
```

Just keeping it consistent with the other number in SetWeather

Reply:
Oops, my bad!

Reply:
Also not sure how correct the 21 claim is, just because it's not favourable weather/bugged, doesn't make it any less valid: http://weedarr.wikidot.com/weather

Reply:
21 is correct since the game has 0 to 20 that's 21 as you pointed out earlier.  Starting 21st weather ID and onwards its OOB and has UB which as mentioned in the #80, so it looks good to me.